.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Create and publish Docker image
+
+on:
+  push:
+    branches:
+    - main
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Thanks for the PR! Is this something that needs to be configured on the repo side?

Reply:
I don't think so. I've done this before and I don't remember it needing anything. I think secrets.GITHUB_TOKEN is provided by default.